hopbox/hopbox_app/templates/register.html

```diff
@@ -13,7 +13,7 @@
     <nav class="navbar navbar-expand-lg navbar-white  bg-white mb-3" style="border-bottom: 3px solid #91B957;">
         <div class="container-fluid">
             <a class="navbar-brand ms-2 fw-bold active" href="/userdashboard">
-                <img src="{% static 'HopBox-logo.png' %}" alt="logo_placeholder" width="340" height="155" class="d-inline-block align-text-top rounded">
+                <img src="static/images/HopBox-logo.png" alt="logo_placeholder" width="340" height="155" class="d-inline-block align-text-top rounded">
```

Comment:
I don't think this is the correct way to render static files -- per the learning platform and the django docs, you need to use the templating engine format (e.g. "{% static 'HopBox-logo.png' %} https://docs.djangoproject.com/en/2.2/howto/static-files/